core/src/test/resources/com/squarespace/template/plugins/platform/f-scarcity-default-hidden.html

```diff
@@ -27,10 +27,10 @@
 {item|product-scarcity}
 
 :OUTPUT
-<div class="product-scarcity" data-variant-attributes="{&quot;Color&quot;:&quot;Red&quot;}" hidden>
-    Only 3 left!
-  </div>
+  <div class="product-scarcity" data-variant-attributes="{&quot;Color&quot;:&quot;Red&quot;}" hidden>
```

Comment:
I am not sure why spacing matters. Some tests are failing and I am not sure why. There might be extra spacing or invisible character I am missing?

Reply:
The language is whitespace-sensitive, so the output needs to match whitespace as well.

Reply:
@phensley I am not sure what I'm missing here. Appreciate any help with this! 

![Screen Shot 2023-07-31 at 10 56 46 AM](https://github.com/Squarespace/template-compiler/assets/88727960/acd2530f-4b38-4796-970e-2eccf6414563)

Reply:
@sizhang12 I'll take a look shortly

Reply:
@phensley  thank you! I will be making the same changes into template-engine as well.